auditlog/views/auditlog_view.xml

```diff
@@ -133,6 +133,7 @@
                             </group>
                             <group colspan="1">
                                 <field name="model_id" readonly="1"/>
+                                <field name="model_name"/>
```

Comment:
Maybe store model.model as well, and show both model_name and model.model iff model_id is False?

Reply:
Similarly, can you show the field name and description below iff field_id is False?

Reply:
model_name is a related field to model_id.model. But I will also store the model_id.name in a separate field and update with your request.